src/themes/teams/components/Chat/chatItemStyles.ts

```diff
@@ -1,7 +1,32 @@
-import { ICSSInJSStyle } from '../../../types'
+import { ICSSInJSStyle, ComponentSlotStylesInput } from '../../../types'
+import { ChatItemProps } from 'src/components/Chat/ChatItem'
+import { ChatMessageVariables } from './chatMessageVariables'
+import { pxToRem } from 'src/lib'
```

Comment:
Keep build green! Change this to:
```jsx
import { pxToRem } from '../../../../lib'
```

Reply:
Yes, we were already discussing this offline (this causes problems in the build too). Will be addressed for sure!

Reply:
fixed